object_detection/g3doc/installation.md

```diff
@@ -0,0 +1,71 @@
+# Installation
+
+## Dependencies
+
+Tensorflow Object Detection API depends on the following libraries:
+
+* Protobuf 2.6
+* Pillow 1.0
+* lxml
+* tf Slim (which is included in the "tensorflow/models" checkout)
+* Jupyter notebook
+* Matplotlib
+* Tensorflow
+
+To install Tensorflow, follow the
+[Tensorflow installation instructions](https://www.tensorflow.org/install/).
```

Comment:
Could you provide a couple of lines for a simple install of TF and direct to the link for others.